src/pages/en/guides/content.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Authoring Content
+description: "Astro is a perfect choice for your content-focused site: blogs, marketing sites, portfolios, and more! Author your content directly in your project, or connect your CMS of choice."
+i18nReady: false
+---
+Astro is a perfect choice for your content-focused site: blogs, marketing sites, portfolios, and more!
+
+Write a blog post directly in Astro using Markdown/MDX, or fetch your content from a headless CMS. Add a layout, create an index of posts and optionally set up an RSS feed to allow readers to subscribe to your posts.
+
+:::tip[Quick Start]
+Want to explore a content-focused Astro site immediately? Open and edit one of our ready-to-go starter templates (blog, portfolio, documentation) right in your browser from [astro.new](https://astro.new)!
+:::
+
+## How to write content for your Site
+
+In Astro, you can author your content in a variety of ways. For example, you can write blog posts:
+
+- Entirely in Markdown, in the editor of your choice.
+- In MDX (`.mdx`) files located in your Astro project.
+- Using a third-part content management system (CMS).
+
+### Markdown Authoring
+Markdown is a convenient syntax for writing large amounts of text with shorthand syntax for some basic HTML formatting such as headers and font styles.
+
+You can use any Markdown editor, including your code editor, to compose your content.
+
+ Create a new `.md` file within your Astro repository, or add a new Markdown file written in an external editor. Some online Markdown editors like [StackEdit](https://stackedit.io/) or [Dillinger](https://dillinger.io) will even sync your work with your Astro repository stored on GitHub.
+
+📚 Learn more about [writing Markdown content in Astro](/en/guides/markdown-content/).
+
+### MDX Authoring
+With the optional Astro MDX integration added to your project, you can include JavaScript expressions and components within your Markdown content. Add UI elements such as a banner or an interactive carousel right in your text, to take your blogging to the next level!
```

Comment:
```suggestion
If you add the MDX integration to your project, you can include JavaScript expressions and components within your Markdown content. Add UI elements such as a banner or an interactive carousel right in your text, or use variables to include dynamic content.
```

(Since this page isn't just about blogging anymore.)